src/services/GroupMeService.ts

```diff
@@ -39,6 +40,23 @@ export class GroupMeService implements MessagingService {
   }
 }
 
+export type GroupMeMention = { loci: [number, number]; type: "mentions"; user_ids: string[] };
```

Comment:
What does a mention attachment look like? Is loci an array of 2 values?

Reply:
yes https://github.com/JosephMart/loki-gm/issues/13#issuecomment-620257474

Reply:
it's actually [[int, int], [int, int]]. `loci.length === user_ids.length`